ern-runner-gen-android/src/AndroidRunnerGenerator.ts

```diff
@@ -75,13 +75,7 @@ function getPackageFilePath(config: RunnerGeneratorConfig): string {
 }
 
 function isOldRunner(config: RunnerGeneratorConfig): boolean {
-  const oldRunner =
-    config.extra &&
-    config.extra.androidConfig &&
-    config.extra.androidConfig.isOldRunner
-      ? true
-      : false
-  return oldRunner
+  return !!config.extra?.androidConfig?.isOldRunner
```

Comment:
Sorry JavaScript noob here, still getting used to this syntax.. Can you quickly explain what this does? ;) `!!` turns null into boolean?

Reply:
Yes you are right :) 
I agree it looks weird, but it's indeed to reduce an expression to a boolean in a less verbose way than 
```javascript
config.extra.androidConfig.isOldRunner	
      ? true	
      : false
```
for example if `config.extra.androidConfig.isOldRunner` evaluates to `null` or `undefined` or `0` or `''`, JS consider these values as "falsy", but they are not of boolean type (and the function signature says it'll return a boolean).
`!!` is just a double negation to make sure that we evaluate the expression to a boolean in any case (it also works if the expression is already evaluating to a boolean).
`!!false` => `!(!false)` => `!(true)` => `false`
`!!0` => `!(!0)` => `!(true)` => `false`

Reply:
Got it. Thanks for explaining!